<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamCalibrateDialog</class>
 <widget class="QDialog" name="CamCalibrateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>637</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Camera Calibration Information</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>551</width>
     <height>201</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>623</width>
     <height>246</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QSpinBox" name="spinBox_calWidth">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>30000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Minimal Matches</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_verboseoutput">
      <property name="text">
       <string>Verbose Output</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="checkBox_useDefaultMinMatches">
      <property name="text">
       <string>Use Default</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="spinBox_numCams">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="stopButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Stop Calibration</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_featureextraction">
      <property name="text">
       <string>Show Feature Extraction</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_minimalMatches">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Number of Cameras</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Calibration Image Height</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_veryVerboseoutput">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Very Verbose Output</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="calibrateCameraButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Run Calibration</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Calibration Images XML</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Calibration Image Width</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_inputXML">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="spinBox_calHeight">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>30000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="recordCalibImagesButton">
      <property name="text">
       <string>Collect New Calibration Images</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
